fury/tests/test_actor.py

```diff
@@ -1,3 +1,4 @@
+from PIL import Image
```

Comment:
Not sure if alphabetical ordering applies to third-party library imports or not

Reply:
I did that initially but it seems that uppercase have priority over lowercase then it is alphabetical.

Reply:
Yes I remember now that the pre-commit made it like this